source/ways-of-working/environments/aks-auto-shutdown.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: AKS Auto Shutdown
+last_reviewed_on: 2023-02-07
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+### Schedule
+
+See below the list of AKS clusters that have auto-shutdown enabled alongside their schedule:
+
+| Shared Services Environment | CFT Environment | Schedule |
+|-|-|-|
+| Demo | Demo | 6:30pm |
+| ITHC | ITHC | 6:30pm |
+| Test | Perftest | 6:30pm |
+| PTLSBOX | PTLSBOX | 6:30pm |
+| Sandbox | Sandbox | 6:30pm |
+
+
+### How to start and stop the cluster from pipeline
+
+If required, an AKS cluster can be started or stopped using a github action on the aks-auto-shutdown repository.
+
+#### Start a cluster
+1. Navigate to [aks-auto-shutdown](https://github.com/hmcts/aks-auto-shutdown) repository.
+2. Navigate to the actions tab.
+3. Select 'AKS Auto Start' from the left hand 'Actions'
+4. Select 'Run Workflow'
+5. Ensure the 'master' branch is selected and select 'Run Workflow'
+6. Select the new build which will appear at the top under Workflow Runs and will be identifable by an orange dot which means the build is currently running.
+7. Select 'build' under 'Jobs'
+8. Expand the 'AKS Auto Start' dropdown to view the job output.
+
+#### Stop a cluster
+1. Navigate to [aks-auto-shutdown](https://github.com/hmcts/aks-auto-shutdown) repository.
+2. Navigate to the actions tab.
+3. Select 'AKS Auto Shutdown' from the left hand 'Actions'
```

Comment:
you could possibly skip a few steps by just direct linking to the right action

Reply:
Good point will take a look :)